### PR TITLE
added dynamic import support

### PR DESCRIPTION
**Pull request by arybitskiy**
- added dynamic import support (with single and double quotes)
- added gitignore
- added try-catch to `fexists`

### CODE REVIEW

File: src/workflow.ts [hunk 1]
```diff
@@ -31,7 +33,7 @@ export async function workflow({ contexts, files }: Api) {
 			});
 
 			if (!importSpecifier) return;
-
+      console.log(filepath, importSpecifier.text());
```
Comment (JakobJingleheimer) [inline src/workflow.ts]:
```suggestion

```

File: src/fexists.ts [hunk 2]
```diff
@@ -9,20 +9,24 @@ export function fexists(
...
 	specifier: Specifier,
 ) {
-	const parentUrl = `${pathToFileURL(dirname(parentPath)).href}/`;
-	const resolvedSpecifier = URL.canParse(specifier)
-		? specifier
-		// import.meta.resolve here is required because we need node's resolution algorithm to
-		// incorporate the results of any hooks that may be helping, such as ones facilitating
-		// tsconfig.compileOptions.paths
-		: fileURLToPath(import.meta.resolve(specifier, parentUrl));
+	try {
```
Comment (JakobJingleheimer) [inline src/fexists.ts]: Why does it need the try/catch? (`access` already handles errors)

**Review by JakobJingleheimer — COMMENTED** [review]
Thanks!

File: src/fexists.ts (hunk 2 above)
Comment (arybitskiy) [reply src/fexists.ts]: the error comes from `import.meta.resolve` - module couldn't be resolved

Comment (JakobJingleheimer) [reply src/fexists.ts]: Ahh, let's wrap only that part then (and include a comment)

Comment (arybitskiy) [reply src/fexists.ts]: done

File: src/fexists.ts [hunk 3]
```diff
@@ -10,12 +10,22 @@ export function fexists(
 	specifier: Specifier,
 ) {
 	const parentUrl = `${pathToFileURL(dirname(parentPath)).href}/`;
-	const resolvedSpecifier = URL.canParse(specifier)
-		? specifier
-		// import.meta.resolve here is required because we need node's resolution algorithm to
+	
+	let resolvedSpecifier: string;
```
Comment (JakobJingleheimer) [inline src/fexists.ts]: I know this ultimately resolves to the same `string` type, but `FSPath` is more descriptive (and not just any string is acceptable).

```suggestion
	let resolvedSpecifier: FSPath;
```

**Comment by JakobJingleheimer** [comment]
I think this is working 🙂 The workflow test snapshot is failing though I think because the existing dynamic import in the e2e fixture is now getting transformed. The workflow test needs to get re-run locally with the update-snapshot flag (and then the updated snapshot file needs to get committed).

I might have time to do late tonight.

**Comment by arybitskiy** [comment]
@JakobJingleheimer update snapshot

**Review by JakobJingleheimer — APPROVED** [review]
W00t! Thanks!

**Pull request merged**